src/error.rs

```diff
@@ -71,6 +72,9 @@ impl fmt::Display for Error {
             ),
             Error::UnexpectedReturn { src_line_number } => {
                 write!(f, "{}: error: unexpected return\n", src_line_number)
+            },
+            Error::InvalidIfStatement => {
+                write!(f, "invalid IF statement, it only supports string-string or numeric-numeric comparisons\n")
```

Comment:
I think we need to add `error: ` prefix here, to be conform with other messages.